exampleSite/config.toml

```
@@ -11,6 +11,7 @@ copyright = "John Doe"
 description = 'Another fine responsive<br />site template freebie<br />crafted by <a href="http://html5up.net">HTML5 UP</a>.<br />Ported to <a href="//gohugo.io">Hugo</a> by Simon Bruder.'
 startbuttonText = "Activate"
 startbuttonLink = "#"
+custom_css = ""
```

I would uncomment this by default and clarify that it accepts a filename (and not a raw css string).
```suggestion
#custom_css = "foo.css"
```